src/styles/setup/variables.scss

```diff
@@ -0,0 +1,69 @@
+@import url('https://fonts.googleapis.com/css?family=Raleway:ital,wght@0,200;0,300;0,900;1,400;1,700&display=swap');
+
+//----------BREAKPOINTS-------------
+
+$bp-xxsmall: 20em; // 320px
```

Comment:
pls, use px for this purposes, because its still very confusing + you will remove comments, and there is no values like 26.2645